sparse/io.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+
+from .coo.core import COO
+
+
+def save_npz(filename, matrix, compressed=True):
+    """ Save a sparse matrix to disk in numpy's ``.npz`` format
+
+    Parameters
+    ----------
+    filename : string or file
+        Either the file name (string) or an open file (file-like object)
+        where the data will be saved. If file is a string or a Path, the
+        ``.npz`` extension will be appended to the file name if it is not
+        already there
+    matrix : COO
+        The matrix to save to disk
+    compressed : bool
+        Whether to save in compressed or uncompressed mode
+
+    Returns
+    -------
+
+    Example
+    --------
+    Store sparse matrix to disk, and load it again:
+
+    >>> import os
+    >>> import sparse
+    >>> import numpy as np
+    >>> dense_mat = np.array([[[0., 0.], [0., 0.70677779]], [[0., 0.], [0., 0.86522495]]])
+    >>> mat = sparse.COO(dense_mat)
+    >>> mat
+    <COO: shape=(2, 2, 2), dtype=float64, nnz=2>
+    >>> sparse.io.save_npz('mat.npz', mat)
+    >>> loaded_mat = sparse.io.load_npz('mat.npz')
+    >>> loaded_mat
+    <COO: shape=(2, 2, 2), dtype=float64, nnz=2>
+    >>> loaded_mat.todense()
+    array([[[0.        , 0.        ],
+            [0.        , 0.70677779]],
+    <BLANKLINE>
+           [[0.        , 0.        ],
+            [0.        , 0.86522495]]])
+    >>> os.remove('mat.npz')
+
+    """
+
+    nodes = {'data':   matrix.data,
+             'coords': matrix.coords,
+             'shape':  matrix.shape}
+
+    if compressed:
+        np.savez_compressed(filename, **nodes)
+    else:
+        np.savez(filename, **nodes)
+
+
+def load_npz(filename):
+    """ Load a sparse matrix in numpy's ``.npz`` format from disk
+
+    Parameters
+    ----------
+    filename : file-like object, string, or pathlib.Path
+        The file to read. File-like objects must support the
+        ``seek()`` and ``read()`` methods.
+
+    Returns
+    -------
+    COO
+        The sparse matrix at path ``filename``
+
+    Example
+    --------
+    Store sparse matrix to disk, and load it again:
+
+    >>> import os
+    >>> import sparse
+    >>> import numpy as np
+    >>> dense_mat = np.array([[[0., 0.], [0., 0.70677779]], [[0., 0.], [0., 0.86522495]]])
+    >>> mat = sparse.COO(dense_mat)
+    >>> mat
+    <COO: shape=(2, 2, 2), dtype=float64, nnz=2>
+    >>> sparse.io.save_npz('mat.npz', mat)
+    >>> loaded_mat = sparse.io.load_npz('mat.npz')
```

Comment:
Same comments as above. Also, it would be best to remove the examples section and in the main description, just add
```
See :obj:`save_npz` for usage examples.
```